vulture_os/applications/logfwd/config/om_elasticsearch.tpl

```diff
@@ -3,7 +3,7 @@
            name="{{output_name}}"
            server={{ servers }}
            searchIndex="{{ template_id }}"
-           template="{{ out_template }}"
+           template="{{ out_template }}_elastic"
```

Comment:
There is currently no template with this suffix, how will the Elasticsearch Log Forwarder work without those templates ?